src/Utility/Text.php

```diff
@@ -26,9 +26,9 @@ class Text
     /**
      * Default transliterator id string.
      *
-     * @param string $defaultTransliteratorId Transliterator identifer string.
+     * @param string $_defaultTransliteratorId Transliterator identifer string.
      */
-    public static $defaultTransliteratorId = 'Any-Latin; Latin-ASCII; [\u0080-\u7fff] remove';
+    protected static $_defaultTransliteratorId = 'Any-Latin; Latin-ASCII; [\u0080-\u7fff] remove';
```

Comment:
Technically this is a BC break as extending classes would fail to find it now.
We should probably not change it to underscore.